snap/validate.go

```diff
@@ -783,12 +783,22 @@ func ValidateApp(app *AppInfo) error {
 	default:
 		return fmt.Errorf(`"refresh-mode" field contains invalid value %q`, app.RefreshMode)
 	}
+	// validate install-mode
+	switch app.InstallMode {
+	case "", "disable":
```

Comment:
as I wrote I think we should also support "enable" which would be the default (same as "")